code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/impl/GithubOAuthImpl.java

```diff
@@ -41,7 +42,7 @@ public Account getAccount(ApiRequest request) {
             return null;
         }
         String token = request.getServletContext().getRequest().getHeader(AUTH_HEADER);
-        if (StringUtils.isEmpty(token)) {
+        if (StringUtils.isEmpty(token) || !token.startsWith(AUTH_TYPE)) {
```

Comment:
And one more pedantic comment from me - if the token is not of an "Bearer" (Github) type, we should return null right away w/o retrieving the token from the request parameters. As this class is only responsible for account lookups for github auth requests.

String token = request.getServletContext().getRequest().getHeader(AUTH_HEADER);
if (token != null && !token.startsWith(AUTH_TYPE)) {
return null
} else if (token == null) {
token = request.getServletContext().getRequest().getParameter("token");
......
}